src/c/dashboard-subscription-card-detail.js

```diff
@@ -109,6 +119,17 @@ const dashboardSubscriptionCardDetail = {
                                     disabledClass: 'a.btn.btn-small.btn-inline.btn-terciary.w-button'
                                 })
                             ])
+                        ]),
+                        m('.u-marginbottom-20.card.u-radius', [
+                            m('.fontsize-small.fontweight-semibold.u-marginbottom-10',
+                                'Endereço'
+                            ),
+                            m('.fontsize-smaller', [
+                                m('div', `${user.address.address_street}, ${user.address.address_number}, ${user.address.address_complement}`),
```

Comment:
This will lead to a trailing comma if there is no complement. You could instead put these into an array and ```join(', ')``` then, for example.